File: cpp/Lessons/sum_series.cpp
#include<iostream>
using namespace std;
int main(){
	double x=1;
	int i;
	for (i = 2; i < 100; i++) {
		x=x+1.0/i;
	}
	cout<<"this sum is"<<x<<endl;
	return 0;
}

File: python/Lab2/sum_series.py
print("program sums the series 1+1/2 +1/3+ .........+1/100 \n")
summ=0
for i in range(1,101):
      summ +=1/i
      if( i%10)==0:  #to print the sum after every to terms.
            print("the summ is ",summ)
print("the final sum is",summ)